cmd/main.go

```diff
@@ -486,7 +482,7 @@ func getOCMClient(log logrus.FieldLogger, metrics metrics.API) *ocm.Client {
 	return ocmClient
 }
 
-func createS3Bucket(objectHandler s3wrapper.API) {
+func createS3Bucket(objectHandler s3wrapper.API, log logrus.FieldLogger) {
```

Comment:
We could move this inside the `S3Client` as it's specific to the S3 Implementation.

Reply:
I had actually suggested the opposite in https://github.com/openshift/assisted-service/pull/1129#discussion_r587725474

Reply:
Oh, I see your reasoning in that comment. Makes sense to me. It does feel that this is not the best place for this function but I also agree with you that keeping the wrapper as close to a client as possible would be better.

Disregard my previous comment, then :D